test/library/packages/Python/correctness/argPassingTest.chpl

```diff
@@ -90,7 +91,7 @@ proc test_three_args_with_default_and_kwargs(mod: borrowed Module) {
   func(NoneType, 8, kwargs=["b" => 10]);
   func(NoneType, 8, kwargs=["c" => 11, "abc" => 12]);
 }
-proc test_varargs_and_kwargs(mod: borrowed Module) {
+proc test_varargs_and_kwargs(mod: borrowed Module, flush: borrowed Value) {
```

Comment:
can't you make flush a global variable instead of threading it through?

Reply:
Yes, but that would require making `interp` a global variable as well

Reply:
That makes me think of two things, in __ascending__ order of quality:

1. Can you make `interp` global :3 
2. What if `interp` had a flush method, so that you didn't have to use a `Value` in user code? Given that you'd need to do this for all interspersed IO, I think it might be worth it.

Reply:
I know this is just a test, but I dislike making `interp` global. However, I do find adding `interp.flush()` appealing. `Value` is definitely a user facing type, but having `interp.flush()` still seems like a useful helper.